book/src/pong-tutorial/pong-tutorial-01.md

```diff
@@ -44,19 +43,16 @@ just implement two methods:
 # use amethyst::renderer::{DisplayConfig, DrawFlat, Pipeline,
 #                          PosTex, RenderBundle, Stage};
 # struct Pong;
-impl<'a, 'b> SimpleState<'a,'b> for Pong {
+impl<'a, 'b> SimpleState<'a, 'b> for Pong {
 }
 ```
 
-The `handle_event` method is executed for every event before updating, and it's
-used to react to events. It returns a `Trans`, which is an enum of state machine
-transitions. In this case, we're watching for the Escape keycode, and the
-`CloseRequested` event from the Window. If we receive it, we'll return
-`Trans::Quit` which will be used to clean up the `State` and close the application.
-All other keyboard input is ignored for now.
-
-The `update` method is executed after events have happened.  In this instance
```

Comment:
Why did you remove this?

Reply:
I think it is because it is not actually used and so feels out of place.
I'd suggest instead introducing it by mentioning that the following method can be implemented, and what they do.

Reply:
This text was written for the `latest` version of the book, and as you can see on https://www.amethyst.rs/book/latest/pong-tutorial/pong-tutorial-01.html, the impl is filled with the mentioned functions there. But in the `master` version, the text just gives the impression that something is missing from the example. Also, I previously understood that the comment from  @torkleyy aims just at the `update` method info having no alternative in my suggested version, so I changed that in the latest commit. If you feel like that the info is unnecessary/confusing there, just let me know and I'll remove it again.